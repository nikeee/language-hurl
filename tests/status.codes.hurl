HTTP

HTTP 200

HTTP/1.1 200
