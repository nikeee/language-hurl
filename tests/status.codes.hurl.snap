>HTTP
#^^^^ source.hurl constant.language.version.hurl
>
>HTTP 200
#^^^^ source.hurl constant.language.version.hurl
#    ^^^^ source.hurl constant.numeric.status.hurl
>
>HTTP/1.1 200
#^^^^^^^^ source.hurl constant.language.version.hurl
#        ^^^^ source.hurl constant.numeric.status.hurl
>